sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/operation/ClassicTableTypeMapping.java

```diff
@@ -69,7 +72,8 @@ public ClassicTableTypeMapping() {
   public String[] mapToHiveType(String clientTypeName) {
     Collection<String> hiveTableType = clientToHiveMap.get(clientTypeName.toUpperCase());
     if (hiveTableType == null) {
-      LOG.warn("Not supported client table type " + clientTypeName);
+      LOG.warn("Not supported client table type {}",
+        MDC.of(LogKeys.CLASSIS_TABLE_TYPE$.MODULE$, clientTypeName));
```

Comment:
Seem there is a typo `CLASSIC`.
Let's simply use `TABLE_TYPE` for all the keys in this file.

Reply:
Done